requirements.in

```diff
@@ -44,7 +44,7 @@ openapi-spec-validator
 #-----------------------------------
 #   Beautiful soup
 #-----------------------------------
-beautifulsoup4==4.12.2
+beautifulsoup4
```

Comment:
Don't we need to mention the version ?

Reply:
You don't need to pin it exactly in `.in` files - in those files we should give broad requirements, and then those get resolved to exact versions in `.txt` files.

Pinning to this exact version made it impossible to bump utils, because [utils was recently made to require 4.12.3](https://github.com/communitiesuk/funding-service-design-utils/commit/2c68a71654699eccebe27b896778ae1f1c29520b) (which I disagree with, tbh - a library should only in very exceptional/unusual circumstances require an exact version of something).